tests/ui/traits/const-traits/const-impl-trait.rs

```diff
@@ -1,4 +1,7 @@
+//@ compile-flags: -Znext-solver
 //@ known-bug: #110395
+//@ failure-status: 101
+//@ dont-check-compiler-stderr
```

Comment:
a little confused about these lines (should indicate an ICE) with `tests/ui/traits/const-traits/const-impl-trait.stderr` also existing

Reply:
just forfor to delete it